File: Ruby/itsasign.rb
a = 1
words = []
openb = ''
while a <= 4
  word = gets.chomp.upcase
  words << word
  a += 1
end

for i in words
  if i == i.reverse
    openb = 'yes'
    break
  end
end

if openb == 'yes'
  print('Open')
else
  print('Trash')
end

File: Python/itsasign.py
a = 1
words = []
openb = ''
while a <= 4:
  word = input()
  words.append(word)
  a += 1

for i in words:
  if i == i[::-1]:
    openb = 'yes'
    break

if openb == 'yes':
  print('Open')
else:
  print('Trash')